generators/app/templates/src/app/components/CustomText/styles.js

```diff
@@ -0,0 +1,51 @@
+import { StyleSheet } from 'react-native';
+
+import { blue, white, green, gray, black, transparent } from '../../../utils/colors';
+import { IS_SMALL_DEVICE } from '../../../utils/constants';
+
+const amountToResize = 2 * IS_SMALL_DEVICE;
+
+export default StyleSheet.create({
+  base: {
+    fontFamily: 'poppins',
```

Comment:
This font is not available in this project! Remove this line to solve the red screen of death 🔴